fiftyone/operators/executor.py

```
@@ -378,31 +378,28 @@ async def resolve_type(registry, operator_uri, request_params):
         :class:`fiftyone.operators.types.Property` of an operator, or None
     """
-    computed_target = target or request_params.get("target", None)
-    computed_request_params = {**request_params, "target": computed_target}
-    operator_uri = request_params.get("operator_uri", None)
-    registry = OperatorRegistry()
-    return await resolve_type(registry, operator_uri, computed_request_params)
+    try:
+        return operator.resolve_type(
+            context, context.request_params.get("target", "inputs")
+        )
+    except Exception as e:
```

Ruff (0.7.0)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)